yarn.lock

```diff
@@ -3949,6 +3949,11 @@ color-name@~1.1.4:
   resolved "https://registry.yarnpkg.com/color-name/-/color-name-1.1.4.tgz#c2a09a87acbde69543de6f63fa3995c826c536a2"
   integrity sha512-dOy+3AuW3a2wNbZHIuMZpTcgjGuLU/uBL/ubcZF9OXbDo8ff4O8yVp5Bf0efS8uEoYo5q4Fx7dY9OgQGXgAsQA==
 
+colorette@^1.2.1:
```

Comment:
There should've been diff to remove chalk too right?

Reply:
No worries handled - https://github.com/webpack/webpack-cli/pull/1710

Reply:
Strange, it was auto generated.

Reply:
probably you removed package from `package.json` directly instead of doing `yarn remove ..`